blog.html

```diff
@@ -0,0 +1,70 @@
+		<a href="dogs.html">Dogs</a> / 
+		<a href="blog.html">Blog</a>
+	</nav>
+	
+<section>
+<h1>Adoptadog Blog</h1>
+
+	<div class="flex-container">
+	  <div>
+		<h2>Traveling With Your Dog</h2>
+		<img src="images/blog-1.jpg" alt="a boy huges a dog image">
+		<p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, conet labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea adqui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernamvoluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+		<p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, conet labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea adqui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernamvoluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+	  </div>
+      
+	  <div>
+		<h2>How To Walk Multiple Dogs</h2>
+		<img src="images/blog-2.jpg" alt="a boy huges a dog image">
+		<p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, conet labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea adqui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernamvoluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+		<p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, conet labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea adqui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernamvoluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+	  </div>
+		
+      <div>
+		<h2>Teach Your Dog To Fetch!</h2>
+		<img src="images/blog-3.jpg" alt="a boy huges a dog image">
+		<p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, conet labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea adqui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernamvoluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+		<p>Uptiusd andesci qui nem aut vendion ectur? Debis que explaut laborenia que doluptur, conet labor abor sant poreperum dio quat que doluptatur aut voluptassim quisciatquam ea adqui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernamvoluptassim quisciatquam ea ad qui con nes cus esere dolut hicto teni solutenis alit ulparume nonseca estorer spernam.</p>
+	 </div>
+     </div>  <!-- for flex-container-->
+ </section>
+
+
+<footer>
+	<img src="images/adoptadog-logo.svg" alt="dog LOGO">
+	<!--
```

Comment:
you can get rid of these comments. The img tags are loading the logo's just fine